test/uk/gov/hmrc/alcoholdutycalculator/controllers/RatesControllerSpec.scala

```diff
@@ -43,37 +43,37 @@ class RatesControllerSpec extends SpecBase {
         rateBandList: Seq[RateBand],
         ratePeriod: YearMonth,
         rateType: RateType,
-        abv: AlcoholByVolume,
         alcoholRegimes: Set[AlcoholRegime]
       ) =>
-        when(mockRatesService.rateBands(any(), any())).thenReturn(rateBandList)
+        if (alcoholRegimes.nonEmpty) {
```

Comment:
I couldn't get the => alcoholRegimes.nonEmpty ==> syntax to work here since we're using scalaCheck stuff in scalaTest I guess.